tutorial/setup/self-hosting.md

```diff
@@ -34,8 +40,18 @@ Everything needed to run PlanarAlly can be found in the `server` folder.
 Make sure to install all dependencies by running `pip install --user -r requirements.txt`.
 (If you are familiar with python, it is strongly advised to create a dedicated venv for PA, but this is not a hard requirement.)
 
+Before the server can be started, you have to build the client.
+This is done by the Node.js package manager *npm* with the following command:  
+`npm i`  
+`npm run build`
+
 To run the server you now simply run `python3 planarserver.py` and your server should start up.
 
+In case you want to install and run the server on the latest status of the development branch, you need to run:  
+`npm i`  
+`npm run serve`  
+`python3 planarserver.py dev`  
+
```

Comment:
Everything is good, just this part is not completely correct.

the `npm run serve` and `python3 planarserver.py dev` combo has nothing to to with the dev branch itself (I know it's confusing).  It starts the server in what is called 'hot reload mode', it's a debug/development mode in which you don't have to rebuild the client for every little change, but that it builds the changes on the fly.

I would describe it as 'development mode with hot module reloading'